1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,33 @@
-function setAlarm() {}
+function setAlarm() {
+  const timeInput = document.querySelector("#alarmSet");
+
+  let timeRemaining = document.querySelector("#timeRemaining");
+  let countdownTime = timeInput.value;
+
+  let minutes = Math.floor(countdownTime / 60);
+
+  let seconds = countdownTime - minutes * 60;
+
+  timeRemaining.innerText = `Time Remaining: 0${minutes}:${seconds}`;
```

Comment:
```suggestion

```

Reply:
I will implement your suggestions. Thank you Vitalina 😊